docs/UserGuide.md

```diff
@@ -248,7 +248,7 @@ The details of the tasks are as follows:
     Completion: Completed
  ```
 
-#### 3.3.3. Delete Task(s)
+#### 3.3.3. <a id="delete-tasks">Delete Task(s)</a>
```

Comment:
Mj, can help make this consistent?
* This header does not tally with the TBC
* Not consistent with other headers like "Add Tasks to the Sprint"

Reply:
Ok I've fixed the inconsistencies.